app/src/main/res/layout/activity_scan_qr.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:orientation="vertical" android:layout_width="match_parent"
+    android:layout_height="match_parent">
+    <FrameLayout
+        android:id="@+id/vFragment_holder"
```

Comment:
Becareful, we are using camel case for IDs. Please, don't merge it with snake case